app/routers/dogfinder.py

```diff
@@ -142,8 +150,9 @@ async def add_document(type: str = Form(...), breed: Optional[str] = Form(None),
             # Create the data object
             data_properties = create_data_properties(dogDocument)
             # Create a uuid based on the filename
-            data_properties["uuid5"] = generate_uuid5({ "filename": dogDocument.filename })
+            data_properties[UUID_FIELD] = generate_uuid5({"filename": dogDocument.filename})
             data_properties["document_embedding"] = dog_embedding
+            data_properties[IS_FOUND_FIELD] = False
```

Comment:
You can just put a default value on the DogDocument to False and you don't need this line